### PR TITLE
fix(lights): improve wing lights

### PR DESCRIPTION
**Pull request by DarlanFernandes**
## Summary of Changes
 This PR changes the wing lights to better match the A320neo.

##Screenshots
 Before: 

Microsoft Flight Simulator Screenshot 2021 12 30 - 19 40 07 23

Microsoft Flight Simulator Screenshot 2022 01 09 - 19 03 45 72

Microsoft Flight Simulator Screenshot 2022 01 27 - 21 20 44 64

Microsoft Flight Simulator Screenshot 2022 01 24 - 18 44 07 09

 After: 

Microsoft Flight Simulator Screenshot 2022 02 09 - 20 32 02 45

Wing

Microsoft Flight Simulator Screenshot 2022 02 10 - 23 16 56 69

Microsoft Flight Simulator Screenshot 2022 02 10 - 23 17 28 50

image

## References
 - Referencing the pictures at:
 - 
https://www.youtube.com/watch?v=RlRSEuGhVH8

https://www.youtube.com/watch?v=7k3YiOQnovM&t=74s

https://twitter.com/aviacaodobrasil/status/1475917313744461826?ref_src=twsrc%5Etfw%7Ctwcamp%5Etweetembed%7Ctwterm%5E1475917313744461826%7Ctwgr%5E%7Ctwcon%5Es1_&ref_url=https%3A%2F%2Faeroin.net%2Festreiam-em-uberaba-mg-as-operacoes-com-jatos-comerciais-pela-azul%2F
 
https://aeroin.net/wp-content/uploads/2021/06/A320neo_Cold_weather_1-1024x713.jpg

image

### CODE REVIEW

**Comment by ErickSharp** [comment]
Can you show before/after screenshots in similar camera angles and ambient lighting? It is hard to gauge the differences you made when the photos are so different in composition.

**Comment by DarlanFernandes** [comment]
> Can you show before/after screenshots in similar camera angles and ambient lighting? It is hard to gauge the differences you made when the photos are so different in composition.

ok, i will send.

**Comment by DarlanFernandes** [comment]
New screenshots on the same view with before and after changes

1 Before = 
Microsoft Flight Simulator Screenshot 2022 02 11 - 14 11 53 37

1 After =
Microsoft Flight Simulator Screenshot 2022 02 11 - 14 10 27 95
 
_____________________________

2 Before =
Microsoft Flight Simulator Screenshot 2022 02 11 - 14 12 37 09

2 After =

Microsoft Flight Simulator Screenshot 2022 02 11 - 14 19 11 34

After changes, the lights illuminate the engines, leading edge of the wing and the sharklet, as in the real aircraft.

**Comment by lolisec** [comment]
Great Job so far on this wip.

**Review by aguther — DISMISSED** [review]

**Comment by bouveng** [comment]
Hi, two issues.

This PR removes the volumetric lights for the wing lt. If that is intended, there is no use for 2 .fx files.

The range and intensity feels overdone.

Don't be afraid to introduce more ligjt soirces to sculpt the lights just right.

File: flybywire-aircraft-a320-neo/effects/LIGHT_A32NX_WingLarge.fx [hunk 1]
```diff
@@ -22,7 +22,7 @@ Drag=0.00, 0.00
 X Particle Velocity=0.00, 0.00
 Y Particle Velocity=0.00, 0.00
 Z Particle Velocity=0.00, 0.00
-X Rotation=0.00, 0.00
+X Rotation=20.00, 20.00
```
Comment (bouveng) [inline flybywire-aircraft-a320-neo/effects/LIGHT_A32NX_WingLarge.fx:25]: Just a note. This rotation applies to both wing lights as they use the same .fx file. It is fine in most cases, but I would prefer to have this information in the `systems.cfg` instead so the light knows nothing about its orientation.

File: flybywire-aircraft-a320-neo/effects/LIGHT_A32NX_WingLarge.fx [hunk 2]
```diff
@@ -65,9 +65,9 @@ MinProjSize=0.2
 
 [LightAttributes.0]
 Type=spot
-Size=0.1
-Range=20
-Intensity=50
+Size=0.8
+Range=100
+Intensity=100
```
Comment (bouveng) [inline flybywire-aircraft-a320-neo/effects/LIGHT_A32NX_WingLarge.fx:70]: Why is the "physical" size of the "bulb" increased x 8, the range by x 5 and the intensity with x 2 (actually x 4 in total when combined/calculated with the volumetric light).

Perhaps the light for the leading edge should be better focused and repositioned. (These light does not need to originate from the modeled light on fuselage, feel free to reposition/add more)

File: flybywire-aircraft-a320-neo/effects/LIGHT_A32NX_WingLargeVol.fx [hunk 3]
```diff
@@ -65,11 +65,11 @@ MinProjSize=0.2
 
 [LightAttributes.0]
 Type=spot
-Size=0.1
-Range=20
-Intensity=50
+Size=0.8
+Range=100
+Intensity=100
```
Comment (bouveng) [inline flybywire-aircraft-a320-neo/effects/LIGHT_A32NX_WingLargeVol.fx:70]: Same as previous comment above.

File: flybywire-aircraft-a320-neo/effects/LIGHT_A32NX_WingLargeVol.fx [hunk 4]
```diff
@@ -65,11 +65,11 @@ MinProjSize=0.2
...
 Softness=0.0
 SpotInner=0
-SpotOuter=22
-Volumetric=1
+SpotOuter=25
+Volumetric=0
```
Comment (bouveng) [inline flybywire-aircraft-a320-neo/effects/LIGHT_A32NX_WingLargeVol.fx:74]: Why are we removing the volumetric effect on these lights? If that is inteded, all the properties of this light could be moved into the regular light. Se previous comment about positioning/adding more lights for sculpting this effect.

**Review by bouveng — CHANGES_REQUESTED** [review]

**Comment by bouveng** [comment]
@DarlanFernandes I can assist you with task if you want help or ideas. =)

**Comment by AdamNZ** [comment]
Sorry to hi-jack this thread - but I couldn't find an open one - and it's kind of related ...

My **NAV** (wingtip) lights are so faint they're nigh-on invisiible. Is there an open issue for this - or should I start one. Apologies (and please remove) if this comment is in the wrong place!

**Comment by bouveng** [comment]
> Sorry to hi-jack this thread - but I couldn't find an open one - and it's kind of related ...
> 
> My **NAV** (wingtip) lights are so faint they're nigh-on invisiible. Is there an open issue for this - or should I start a new one? Apologies (and please remove) if this comment is in the wrong place!

@AdamNZ Please open a new ticket for that. Providing screenshots perhaps. NAV LT have never been stronger as they are now, so I don't really understand.

Br.

/J

**Comment by AdamNZ** [comment]
OK - will do (thanks) ... and if anyone has the right permissions, please remove these out of context comments of mine ;-)

**Comment by beheh** [comment]
@DarlanFernandes Any objection if I move this PR back to a draft until you can make the changes that have been suggested?

**Comment by beheh** [comment]
I have moved this PR back to draft so the suggestions from bouveng can be addressed.

**Comment by aguther** [comment]
I'm closing the PR for now due to inactivity. It can be reopened at any time if you want to continue the work on it.

**Pull request closed without merging**